src/components/Login/Login.jsx

```diff
@@ -6,7 +6,7 @@ import Container from '@mui/material/Container';
 import Paper from '@mui/material/Paper';
 import { useRedirectUrl } from '../../hooks';
 import { SOLID_IDENTITY_PROVIDER } from '../../utils';
-import AppHeader from '../AppHeader';
+import NavBar from '../NavBar/NavBar';
```

Comment:
I think we can refactor these components into their own index.js inside their directory. We could leave this for now for a future PR.

Reply:
Noted

Reply:
Given that you've already added the index.js back in, I think you should import this properly as:

`import { NavBar } from './NavBar'`

Instead of what you're using now.